src/features/receive/Receive.tsx

```diff
@@ -59,6 +59,7 @@ export const Receive = (props: IReceiveProps): JSX.Element => {
     transactions,
   } = props
 
+  const [currentTab, setCurrentTab] = useState(0)
```

Comment:
Consider using typescript enum instead of a number. Because when I see `currentTab === 0 ` it's harder to see what is this for, better to have `currentTab === Tab.shielded`. Or ETab for enum since you prefer T for types.

Reply:
Fixed